stackit/internal/services/ske/cluster/datasource.go

```diff
@@ -296,6 +296,21 @@ func (r *clusterDataSource) Schema(_ context.Context, _ datasource.SchemaRequest
 							},
 						},
 					},
+					"dns": schema.SingleNestedAttribute{
+						Description: "DNS extension configuration",
+						Computed:    true,
```

Comment:
```suggestion
						Optional:    true,
```

Reply:
It's also `Computed` for ACL and argus. So what should it be?

Reply:
oh yes, you are right. In datasource, it should be computed.